### PR TITLE
New branch menus

### PR DESCRIPTION
**Pull request by PavelLaptev**
## ☕️ Reasoning

- Align UI/UX of branch buttons and actions
- Improve `ContextMenu` — detect if not enough space to display it

https://github.com/user-attachments/assets/ae484b16-eb01-41a5-aff6-3587d508cc50

### CODE REVIEW

**Comment by krlvi** [comment]
@PavelLaptev looks nice. Just a question, is the "+" button for creating a dependent branch behind the feature flag?

**Comment by PavelLaptev** [comment]
> @PavelLaptev looks nice. Just a question, is the "+" button for creating a dependent branch behind the feature flag?

yep. it has `{#if stackingFeatureMultipleSeries}`

File: apps/desktop/src/lib/branch/StackingSeriesHeader.svelte [hunk 1]
```diff
@@ -121,18 +125,60 @@
...
+	hasPr={!!$pr}
+	openPrDetailsModal={handleOpenPR}
+	reloadPR={handleReloadPR}
+	onopen={() => (contextMenuOpened = true)}
+	onclose={() => (contextMenuOpened = false)}
+/>
+
+<div role="article" class="branch-header" oncontextmenu={(e) => e.preventDefault()}>
+	<PopoverActionsContainer class="branch-actions-menu" stayOpen={contextMenuOpened}>
+		{#if stackingFeatureMultipleSeries}
```
Comment (ndom91) [inline apps/desktop/src/lib/branch/StackingSeriesHeader.svelte]:
```suggestion
		{#if $stackingFeatureMultipleSeries}
```

**Review by ndom91 — APPROVED** [review]
LGTM!

**Review by krlvi — APPROVED** [review]

**Pull request merged**